### DISCUSSION TITLE
Minor Launching Issue

### DISCUSSION THREAD

**Discussion by notkidgamer23** [ikemen-engine/Ikemen-GO#825] [Feature Request]
Not a big problem, but a problem worth noting. If you have the game installed anywhere that requires admin perms to run, like program files for example the game won't ask for permission and simply won't do anything. It however works fine when run as admin. Again, very minor issue.

**Comment by Windblade-GR01** [comment]
That's a common on all programs, not sure if GO can check folder permissions before launching.
If it can we could made it ask for admin permission when the folder requires it.
